CONTRIBUTING.rst

```diff
@@ -22,6 +22,15 @@ You can now clone your fork of scVelo and install the development mode
     git clone https://github.com/YOUR-USER-NAME/scvelo.git
     cd scvelo
     git checkout --track origin/develop
+    
+
+Unix Systems
+^^^^^^^^^^^^^^^
```

Comment:
I know it's just style but could you please make the `^` align with the text, so
```
Unix Systems
^^^^^^^^^^
```

instead of

```
Unix Systems
^^^^^^^^^^^^^^^
```?